src/packages/protoc-gen-openapiv2/update-hash.py

```
@@ -0,0 +1,220 @@
+            
+            # Write modified content
+            with open(nix_file, 'w', encoding='utf-8') as f:
+                f.write(content)
+            
+            # Try to build, which will fail and show the correct vendor hash
+            result = subprocess.run(
+                ['nix-build', '-E', f'(import <nixpkgs> {{}}).callPackage ./{NIX_FILE} {{}}'],
+                cwd=Path(__file__).parent,
+                capture_output=True,
+                text=True
+            )
```

Pylint (3.3.7)

[warning] 'subprocess.run' used without explicitly defining the value for 'check'.

(W1510)